test/integration/targets/wait_for/files/write_latin1.py

```diff
@@ -0,0 +1,11 @@
+from __future__ import (absolute_import, division, print_function)
+__metaclass__ = type
+
+import sys
+
+# latin1 encoded bytes
+# to ensure wait_for doesn't have any encoding errors
+data = b'premi\xe8re is first\npremie?re is slightly different\n????????? is Cyrillic\n? am Deseret\n'
```

Comment:
Are the `?` supposed to be literal question marks?

Reply:
Yes, kind of.  I sourced this from another pre-existing integration test script.  The 2 scripts match in content, so it's nothing lost.  The content still triggers the issue, and ensures we don't regress.

Reply:
Addressed